stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/WebSocketConstants.java

```diff
@@ -72,6 +72,7 @@
     public static final String RESOURCE_NAME_UPGRADE = "upgrade";
 
     public static final String WEBSOCKET_HANDSHAKER = "WEBSOCKET_MESSAGE";
+    public static final String MESSAGE_DATA_SOURCE = "message_datasource";
```

Comment:
What is this `message_datasource` for?

Reply:
In the entity, the payload is set by the `message_datasource` key as native data. So, I use it to [get the response's payload](https://github.com/ballerina-platform/ballerina-lang/pull/24843/files#diff-c36bff13846589a8cf5f4454fff28d25R167) from the entity.